src/utils/css-utils.js

```diff
@@ -5,6 +5,7 @@ const sizes = {
     desktop: 1200,
     tablet: 768,
     phone: 599,
+    verySmall: 280,
```

Comment:
Actually the idea was to use mobile first approach, so we should consider all sizes less than tablet - mobile